lib/vtls/vtls.h

```diff
@@ -43,7 +43,7 @@ struct Curl_ssl_session;
 #define VTLS_INFOF_NO_ALPN                                      \
   "ALPN: server did not agree on a protocol. Uses default."
 #define VTLS_INFOF_ALPN_OFFER_1STR              \
-  "ALPN: offers %s"
+  "ALPN: client offers %s"
```

Comment:
In the name of clarify, should it perhaps even say `curl offers...` ?

Reply:
done